### PR TITLE
added run command

### PR DESCRIPTION
**Pull request by NivRichter**
Added docs for running command on user codespace

### CODE REVIEW

**Comment by shem8** [comment]
You can see here an example:
https://github.com/trywilco/wilco-engine/pull/2387

File: docs/building-your-quest/actions/run-command.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+### Description
+
+**Type**: action
+
+Run a command on the user's codespace. The action uses an agent running on the remote machine.
+
+## Params
+
+- **command:** The command to run on the remote machine. i.e. `ls -l`
+- **timeout**: The maximal time in seconds to wait for the command to finish. The default is 60 seconds.
```
Comment (shem8) [inline docs/building-your-quest/actions/run-command.md:22]: 30 seconds, I think that makes more sense

**Review by shem8 — APPROVED** [review]

File: docs/building-your-quest/actions/run-command.md (hunk 1 above)
Comment (NivRichter) [reply docs/building-your-quest/actions/run-command.md]: > 30 seconds, I think that makes more sense

changed

Comment (shem8) [reply docs/building-your-quest/actions/run-command.md]: approved

**Pull request merged**